src/core/Table/TableCell.tsx

```diff
@@ -0,0 +1,87 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { Cell, CellProps, TableInstance } from 'react-table';
+import cx from 'classnames';
+import { getCellStyle } from './utils';
+import EditableCell, { OnCellEditCallbackParams } from './EditableCell';
+import SubRowExpander from './SubRowExpander';
+import { SELECTION_CELL_ID } from './hooks';
+
+export type TableCellProps<T extends Record<string, unknown>> = {
+  cell: Cell<T>;
+  cellIndex: number;
+  onCellEdit?: (cellEditParams: OnCellEditCallbackParams<T>) => void;
+  isRowDisabled: boolean;
```

Comment:
`isDisabled` would make more sense, no? Why do we care about row?

Reply:
Renamed

Reply:
Changed to just params